lib/req/test.ex

```diff
@@ -273,15 +279,21 @@ defmodule Req.Test do
     end
   end
 
+  @deprecated "Use `fetch_stub!/1` instead."
+  def stub(name) do
+    fetch_stub!(name)
+  end
+
   @doc """
-  Returns the stub created by `stub/2`.
+  Fetches the stubbed request handler (plug) for the given `name`.
   """
-  @spec stub(stub()) :: term()
-  def stub(stub_name) do
-    case NimbleOwnership.fetch_owner(@ownership, callers(), stub_name) do
+  @doc since: "0.5.0"
+  @spec fetch_stub!(stub()) :: plug()
```

Comment:
wdyt about just fetch!?

```suggestion
  @spec fetch!(stub()) :: plug()
```

Reply:
Mh I don't think it works. I don't think `fetch_stub!` works very well either. `Req.Test.fetch!` is like... fetching what?! Maybe `Req.Test.fetch_for_name!`? I don't think anyone is expected to use this really, so I’m not sure we need a nice name, but I’m open to other ideas as well.

Reply:
right, let's defp it then?

Reply:
Yeah let's do that for now.